app/src/main/java/com/ichi2/apisample/MainActivity.java

```diff
@@ -52,6 +52,8 @@
 
     private static final String STATE_REF_DB = "com.ichi2.apisample.uistate";
 
+    private Map<String, String> fieldLabels;
```

Comment:
It looks like this variable could be defined as final and be initialized right here